### PR TITLE
BP-85: 2023 migration

### PR DESCRIPTION
**Pull request by rkennedy1**

### CODE REVIEW

File: Server/src/routes/playerRoute.ts [hunk 1]
```diff
@@ -70,10 +71,10 @@ playerRoute.get(
     let year = req.params['year'];
     let currentVersion = await getCurrentVersion(year);
     let playerIdRange = getTeamIdRange(year);
-    let query = `SELECT * FROM players WHERE playerId > ${playerIdRange.lowerBound} AND playerId < ${playerIdRange.upperBound} AND version > ${version}`;
```
Comment (Mskennedy67) [inline Server/src/routes/playerRoute.ts:73]: we should get rid of the select * in future

File: Utilities/loadPicksFromExcel.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+def parse_excel(file_path):
+    try:
+        workbook = openpyxl.load_workbook(file_path)
+        sheet = workbook.active
+        players = insertPlayers(sheet)
+        row_count = sheet.max_row
+
+        for row in range(2, row_count, 2):
+            bowlId = sheet.cell(row=row + 1, column=1).value
+            # top = {
```
Comment (Mskennedy67) [inline Utilities/loadPicksFromExcel.py:72]: clean up comments

File: Web/src/components/Homepage.tsx [hunk 3]
```diff
@@ -71,7 +55,7 @@ export function Homepage() {
         </Grid>
         <Grid item xs={12}>
           <BowlpoolTable
-            year={2022}
+            year={2023}
```
Comment (Mskennedy67) [inline Web/src/components/Homepage.tsx:58]: maybe put in env

**Review by Mskennedy67 — APPROVED** [review]
🥇 approved

File: Server/src/routes/playerRoute.ts (hunk 1 above)
Comment (rkennedy1) [reply Server/src/routes/playerRoute.ts:73]: created a jira: [BP-86](https://rskennedy.atlassian.net/browse/BP-86)

File: Web/src/components/Homepage.tsx (hunk 3 above)
Comment (rkennedy1) [reply Web/src/components/Homepage.tsx:58]: created a jira: [BP-87](https://rskennedy.atlassian.net/browse/BP-87)

**Pull request merged**